packages/patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js

```diff
@@ -830,3 +830,4 @@ setTimeout(function timeoutFunction() {// This prevents GT elements get null.
     });
   }
 }, 1000);// When it's less than 1000, the iframe is null in full screen display.
+}
```

Comment:
@clairesunstudio This curly bracket closes this if statement: https://github.com/massgov/mayflower/pull/1696/files#diff-7b3a2bd65c30faae06061f8aecab1e5e30c7b2a796b16c5e6b2727afbd1f8f78R34. This includes function declarations as well. Is that intentional?

Reply:
yes, that's intentional, actually, I'm going to also include all the declaration queries. 

We need to figure out a better way to scope JS, many JS issues I see are related to it not being scoped properly.